package/configuration-magento2-gcp/magento-gcp.yaml

```diff
@@ -0,0 +1,706 @@
+      - type: FromCompositeFieldPath
+        fromFieldPath: status.providers.helm.name
+        toFieldPath: spec.providerConfigRef.name
+
+
+    - name: magento-release
+      base:
+        apiVersion: helm.crossplane.io/v1beta1
+        kind: Release
+        metadata:
+          annotations:
+            crossplane.io/external-name: "magento"
+        spec:
+          forProvider:
+            chart:
+              name: magento
+              repository: oci://registry-1.docker.io/bitnamicharts
+              version: "25.5.0"
+            namespace: default
+            values:
+              persistence: 
+                enabled: false
+              magentoHost: ""
+              magentoPassword: ""
+              externalDatabase:
+                host: ""
+                port: 3306
+                user: ""
+                password: ""
+                database: ""
+              mariadb:
+                enabled: false
+              image:
+                debug: true
+              ingress:
+                ingressClassName: nginx
+                enabled: true
+                hostname: ""
+          providerConfigRef:
+            name: ""
+      patches:
+      - type: FromCompositeFieldPath
+        fromFieldPath: status.providers.helm.name
+        toFieldPath: spec.providerConfigRef.name
+      - type: FromCompositeFieldPath
+        fromFieldPath: status.providers.helm.name
+        toFieldPath: spec.providerConfigRef.name
+      - type: FromCompositeFieldPath
+        fromFieldPath: spec.database.password
+        toFieldPath: spec.forProvider.values.externalDatabase.password
+      - type: FromCompositeFieldPath
+        fromFieldPath: spec.database.password
+        toFieldPath: spec.forProvider.values.magentoPassword
+      - type: FromCompositeFieldPath
+        fromFieldPath: status.sqlInstance.user.name
+        toFieldPath: spec.forProvider.values.externalDatabase.user
+      - type: FromCompositeFieldPath
+        fromFieldPath: status.sqlInstance.database.name
+        toFieldPath: spec.forProvider.values.externalDatabase.database
+      - type: FromCompositeFieldPath
+        fromFieldPath: status.sqlInstance.host
+        toFieldPath: spec.forProvider.values.externalDatabase.host
+      - type: FromCompositeFieldPath
+        fromFieldPath: spec.magento.hostname
+        toFieldPath: spec.forProvider.values.magentoHost
+      - type: FromCompositeFieldPath
+        fromFieldPath: spec.magento.hostname
+        toFieldPath: spec.forProvider.values.ingress.hostname
+
+
+    - name: db-instance-mysql
+      base:
+        apiVersion: sql.gcp.upbound.io/v1beta1
+        kind: DatabaseInstance
+        spec:
+          forProvider:
+            databaseVersion: MYSQL_5_7
+            deletionProtection: false
+            region: ""
+            settings:
+             - ipConfiguration:
+               - privateNetworkRef:
+                  name: ""
+            settings:
+              - diskSize: 20
```

Comment:
@ca7alindev Add GCP persistent disc creation by current composition and use it here

Reply:
@evghen1, persistent disk for databaseInstance?